app/templates/buyers/index.html

```diff
@@ -0,0 +1,60 @@
+{% extends "_base_page.html" %}
+{% block page_title %}Your account - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+    {
+        "link": "/",
+        "label": "Digital Marketplace"
+    }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+<div class="grid-row">
+  <div class="column-full">
+    {% with
+        context = current_user.email_address,
+        heading = current_user.name
+    %}
+    {% include 'toolkit/page-heading.html' %}
+    {% endwith %}
+  </div>
+  <div class="column-full dm-account-overview-options">
+    
+    <div class="framework">
+      <div class="dmspeak">
+        <h3 class="heading-xmedium">Cloud hosting, software and support</h3>
+      </div>
+      <p>Buy cloud technology through the G-Cloud framework.</p>
```

Comment:
This content is weird, but it's certainly fine for us to test with

Reply:
Cloud hosting software and support should be pulled from frameworks

Reply:
Yep, ideally no frameworks should be explicitly named in our front-end apps.

Reply:
Apps need to be applicable to any future similar frameworks.

Reply:
`This content is weird` :+1:
For years we have tried to remove the word "framework" as much as possible from the marketplace, the rationale being that buyers come to buy a thing and shouldn't need to know or understand the concept of frameworks.  Instead we should help them find the thing they want to buy.

Unless there has been a complete U-turn in policy around this for the marketplace in general then I'd be wary of even putting this in front of people in labs.

Reply:
We don't have @karlchillmaid available right now, so this will have to go to labs as it is. We'll keep the ticket open until the content is sorted.